module-3/pop/HomePage.js

```diff
@@ -3,20 +3,27 @@
  * Add main widgets and element of the page and write tests
  * for it (test/pop/HomePage.spec.js).
  */
-const Layout = require('./Layout');
-const Element = require('./Element');
-const Elements = require('./Elements');
+const Layout = require("./Layout");
+const Element = require("./Element");
 
 module.exports = class HomePage extends Layout {
-    constructor() {
-        super('/');
+  constructor() {
+    super("EPAM Home Page", "https://www.epam.com/", { css: "body" });
 
-        this.header = new Element('Header', by.css('header'));
```

Comment:
Could you please add a bigger component? e.g.: header, and please add a children to it.